### PR TITLE
Add Location column to CLI skills table

### PR DESCRIPTION
**Pull request by umago**
## Summary

There are three types of skills:

* Project -> Skills under the current working directory (e.g. .agents/skills/, .goose/skills/).

* Global -> Skills loaded from outside the working directory (e.g. ~/.agents/skills/, ~/.config/agents/skills/).

* Builtin -> Skills that ship with goose (BuiltinSkill kind).

This helps users distinguish project-specific skills (shared with the team via version control) from globally installed ones (personal to their machine) and built-in ones (shipped with goose, not editable), making it clear which skills are portable, which are personal, and which are part of goose itself.

### Screenshots/Demos (for UX changes)
```
🪿 /skills
+---------------------+----------+------------------------------------------------------------------------------------------------------------------------------------------+
| Skill               | Location | Description                                                                                                                              |
+===========================================================================================================================================================================+
| code-review         | project  | Senior engineer code review focused on catching issues before they become PR comments. Reviews only changed lines, categorizes issues by |
|                     |          | priority, and fixes them one by one. Use when the user says "code review", "review my code", "review this branch", or wants pre-PR       |
|                     |          | feedback.                                                                                                                                |
|---------------------+----------+------------------------------------------------------------------------------------------------------------------------------------------|
| coding              | global   | Strict coding guidelines biasing toward caution, precision, and minimalism. Use for all coding tasks to ensure clean, surgical changes.  |
|---------------------+----------+------------------------------------------------------------------------------------------------------------------------------------------|
| create-app-e2e-test | project  | Create app e2e tests for the Goose desktop app using the Tauri app test driver. Use when the user wants to generate, write, or verify UI |
|                     |          | tests that run against the live app.                                                                                                     |
|---------------------+----------+------------------------------------------------------------------------------------------------------------------------------------------|
| create-pr           | project  | Create a GitHub PR from the current branch: handle uncommitted changes, generate a summary, and submit via gh CLI. Use when the user     |
|                     |          | says "create PR", "open PR", "submit PR", "push PR", or wants to create a pull request.                                                  |
|---------------------+----------+------------------------------------------------------------------------------------------------------------------------------------------|
| edge-case-finder    | project  | Analyzes branch changes to find edge cases, error states, and untested user flow paths in UI code. Use when the user says "find edge     |
|                     |          | cases", "what am I missing", "edge cases", "test my flows", "what could go wrong", or wants to harden a feature before shipping.         |
|---------------------+----------+------------------------------------------------------------------------------------------------------------------------------------------|
| goose-doc-guide     | built-in | Reference goose documentation to create, configure, or explain goose-specific features like recipes, extensions, sessions, and           |
|                     |          | providers. You MUST fetch relevant goose docs before answering. You MUST NOT rely on training data or assumptions for any goose-specific |
|                     |          | fields, values, names, syntax, or commands.                                                                                              |
|---------------------+----------+------------------------------------------------------------------------------------------------------------------------------------------|
| insight             | global   | Direct, casual, and focused on mental models and analogies.                                                                              |
|---------------------+----------+------------------------------------------------------------------------------------------------------------------------------------------|
| peer-review         | global   | Conducts rigorous technical audits from a Senior Engineer perspective to identify logic flaws, security vulnerabilities, and             |
|                     |          | architectural debt.                                                                                                                      |
|---------------------+----------+------------------------------------------------------------------------------------------------------------------------------------------|
| repo-scan           | global   | Guidelines for analyzing an unfamiliar codebase. Prioritizes high-level architecture, structural clarity, and providing entry points for |
|                     |          | deep-dives.                                                                                                                              |
|---------------------+----------+------------------------------------------------------------------------------------------------------------------------------------------|
| sys-design          | global   | Architect-level skill for generating a Technical Design Doc (TDD) intended as the primary input for coding agents and human review.      |
+---------------------+----------+------------------------------------------------------------------------------------------------------------------------------------------+
```

### CODE REVIEW

File: crates/goose-cli/src/session/mod.rs [hunk 1]
```diff
@@ -919,13 +920,24 @@ impl CliSession {
...
 
         let mut sorted_skills = skills;
         sorted_skills.sort_by(|a, b| a.name.cmp(&b.name));
 
         for skill in &sorted_skills {
-            table.add_row(vec![Cell::new(&skill.name), Cell::new(&skill.description)]);
+            let location = if skill.source_type == SourceType::BuiltinSkill {
+                "built-in"
+            } else if skill.global {
+                "global"
```
[the remark this answers is not in the document]
Comment (umago) [reply crates/goose-cli/src/session/mod.rs:932]: Well, first this is something outside of this PR scope because if "global" is being set incorrectly, it should be fixed in another PR. But also, starting goose from `$HOME`  is a super edge case and really not how goose should be used.

**Review by DOsinga — APPROVED** [review]
Nice one — this is genuinely useful! Showing the location at a glance saves a real head-scratch when you're wondering why a skill is (or isn't) picked up. Thanks for the clean implementation and for addressing the review feedback. 🎉

**Comment by umago** [comment]
Thanks for the review @DOsinga!
